src/components/LocaleSelector.vue

```diff
@@ -12,14 +12,15 @@
   <div class="locale-selector">
     <select
       v-model="$i18n.locale"
+      @change="updateRouter"
       :aria-label="$t('select-language')"
     >
```

Comment:
`v-model` and the `change` event is probably not the best. Lets do the assignment inside the change event?

```suggestion
      :value="$i18n.locale"
      :aria-label="$t('select-language')"
      @change="updateRouter"
    >
```

Reply:
Good idea, fixed it